### DISCUSSION TITLE
How to use mask feature to extract clothes from given pictures

### DISCUSSION THREAD

**Discussion by FurkanGozukara** [community/community#47232] [Programming Help]
### Select Topic Area

Question

### Body

Hello. I am trying to extract clothes from given picture images

How can I do that?

so far this is what I made

input image

![a](https://user-images.githubusercontent.com/19240467/218445077-b4554d10-afe8-4433-a87b-f7c343071d10.jpg)

run command

!mask "C:\Users\King\Downloads\a.jpg" -tm "clothes"

outputs

![000031 a clothes deselected](https://user-images.githubusercontent.com/19240467/218445119-057481da-2a13-408a-85d5-cad54ef2696f.png)
![000031 a clothes masked](https://user-images.githubusercontent.com/19240467/218445127-6025e7f8-51b7-444e-bf71-715b16a8576e.png)
![000031 a clothes selected](https://user-images.githubusercontent.com/19240467/218445130-56778e61-13ce-49b8-9195-4c3542327f14.png)

expected output

ty so much

![image](https://user-images.githubusercontent.com/19240467/218445178-1a1924ee-5620-4745-93b9-bb2e551c5f22.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @FurkanGozukara! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
